### PR TITLE
Fixes fake buttons on the push.bdnewsx.com

### PR DESCRIPTION
**Pull request by wout-codes**
Fake button and useless things.
Link: `https://push.bdnewsx.com/demo`

Here are the necessary screenshots https://github.com/AdguardTeam/AdguardFilters/pull/75108#issuecomment-781201007

### CODE REVIEW

**Comment by zloyden** [comment]
> Fake button and useless things.
> Link: `https://push.bdnewsx.com/demo`

Unfortunately, I got immediately redirect to the `techrfour.com`. For such requests without task you should provide atleast a screenshot.

**Comment by wout-codes** [comment]
> > Fake button and useless things.
> > Link: `https://push.bdnewsx.com/demo`
> 
> Unfortunately, I got immediately redirect to the `techrfour.com`. For such requests without task you should provide atleast a screenshot.

@zloyden Yep, they redirects always. By the way, there are placeholders in that site.

**Comment by zloyden** [comment]
> By the way, there are placeholders in that site.

Fixed for `techrfour.com`. What about your rule `push.bdnewsx.com##.banner`?

**Comment by wout-codes** [comment]
@zloyden Placeholders here: `https://techymozo.com/5-facts-about-mobile-apps/`

**Comment by wout-codes** [comment]
@zloyden 
image

**Comment by zloyden** [comment]
> I also added `techymozo.com` because of leftovers.

It was fixed. Please also use the spoiler for the screenshots.

**Comment by wout-codes** [comment]
> > I also added `techymozo.com` because of leftovers.
> 
> It was fixed. Please also use the spoiler for the screenshots.

Ok.

**Comment by wout-codes** [comment]
@zloyden You can merge this

**Comment by zloyden** [comment]
> You can merge this

Please read [this](https://github.com/AdguardTeam/AdguardFilters/pull/75108#issuecomment-781121129).

**Comment by wout-codes** [comment]
@zloyden 
<details><summary>Screenshot:</summary>

image
</details><br/>

**Comment by zloyden** [comment]
So are we talking about `Click here to download`? Can you show the element with opened devtools?

**Comment by wout-codes** [comment]
@zloyden 

<details><summary>Screenshot:</summary>

! With the rules I added: 
image
</details><br/>

<details><summary>Screenshot:</summary>

! Without the rules I added:
image
</details><br/>

<details><summary>Screenshot:</summary>

! DevTools:
image
</details><br/>

**Comment by zloyden** [comment]
Ok, thanks. I edited your first post by adding a link to the screenshots. Please do the same in the future.

**Comment by wout-codes** [comment]
> Ok, thanks. I edited your first post by adding a link to the screenshots. Please do the same in the future.

Okay.

File: EnglishFilter/sections/specific.txt [hunk 1]
```diff
@@ -224,7 +224,7 @@ xxxparodyhd.net,movgotv.com,123kmovies.net,123movies.*,bluemovies.co,f5movies.to
...
+earncoin.site,push.bdnewsx.com,rb7url.net,teenlolly.com,best18porn.com,fotodovana.com,1shorten.com,shortlinkpay.com,dextools.io,alternativeto.net,nakedneighbour.com,viral-topics.com,recipestutorials.com,tawiia.com,18tube.xxx,18porno.tv,youx.xxx,xz2.xyz,caixinglobal.com,bitcoinly.in,fakemail.net,internewstv.com,trinddz.com,omni.se,earnguap.com,forums.sonarr.tv,cashurl.in,abdeo8.com,xfantazy.com,printscreenshot.com,cut.my.id,photoeditor.com,9c717baaf805a8436afd7912039e826c.link,uflash.tv,boxlink.us,shortbitsfree.net,hellomagazine.com,adsme.cc,clickurls.site,planetanime.co,rocketfiles.com,pornwhite.com,aplayer2.me,viralizeit.net,sex3.com,softonic.com,3pornstarmovies.com,govtsmartjob.com,cutearn.ca,dfporn.net,cll.press,hellporno.net,youramateurporn.com,getlink.pw,embedtvseries.com,keygames.com,uclick.in,flv2mp3.by,doodhwali.com,smutindia.com,dvdcover.com,xxxadultphoto.com,gabicoins.com,qpdownload.com,windows10portal.com,2conv.com,javcloud.com,underhentai.net,cutwin.com,hqasiananal.com,bradsknutson.com,flv2mp3.org,subscene.com,yourlink.in,iplayer.pw,qpornx.com,sofifa.com,jawcloud.co,indiansgetfucked.com,medi-bayreuth.de,oxforddictionaries.com,tube18.sex##.banner
```
Comment (zloyden) [inline EnglishFilter/sections/specific.txt:227]: One last thing. `push.bdnewsx.com` should be at the beginning.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/specific.txt (hunk 1 above)
Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:227]: Ok added.

**Review by zloyden — APPROVED** [review]

**Pull request merged**